srcpkgs/yt-dlp/patches/nodejs.patch

```diff
@@ -0,0 +1,20 @@
+until deno is packaged, make node the default
+
+--- a/yt_dlp/options.py
++++ b/yt_dlp/options.py
+@@ -464,13 +464,13 @@
+         callback=_list_from_options_callback,
+         type='str',
+         callback_kwargs={'delim': None},
+-        default=['deno'],
++        default=['node'],
+         help=(
+             'Additional JavaScript runtime to enable, with an optional location for the runtime '
+             '(either the path to the binary or its containing directory). '
+             'This option can be used multiple times to enable multiple runtimes. '
+             'Supported runtimes are (in order of priority, from highest to lowest): deno, node, quickjs, bun. '
+-            'Only "deno" is enabled by default. The highest priority runtime that is both enabled and '
++            'Only "node" is enabled by default. The highest priority runtime that is both enabled and '
+             'available will be used. In order to use a lower priority runtime when "deno" is available, '
+             '--no-js-runtimes needs to be passed before enabling other runtimes'))
```

Comment:
Does this mean users don't need to add `--js-runtimes node`? (this is preferable)

Similarly, if yt-dlp is installed from xbps and deno is installed externally,  the user has to use `--js-runtimes deno`? (if this is true, then it's kind of weird as it behaves differently)